packages/react/src/components/Select/Select.tsx

```diff
@@ -222,12 +224,27 @@ const Select = React.forwardRef(function Select(
     ariaProps['aria-describedby'] = helper ? helperId : undefined;
   }
 
+  const selectDefaultTitle = (defaultValue) => {
+    const selectElement = document.getElementById(
+      id
+    ) as HTMLSelectElement | null;
+    if (defaultValue) {
+      const defaultOption: HTMLOptionElement | null | undefined =
+        selectElement?.querySelector(`option[value=${defaultValue}]`);
```

Comment:
Is there a way to declaratively get the value from `children` instead of using this imperative selector?

Reply:
Hi @tay1orjones, removed the imperative selector as I realized it was not needed since we could get value through `selectedIndex`